{"name": "使用无效凭证登录", "status": "passed", "steps": [{"name": "打开登录页面", "status": "passed", "start": 1743469759290, "stop": 1743469759715}, {"name": "输入错误的用户名和密码", "status": "passed", "start": 1743469759715, "stop": 1743469759893}, {"name": "验证错误消息", "status": "passed", "start": 1743469759893, "stop": 1743469759914}], "start": 1743469759290, "stop": 1743469759915, "uuid": "112d383c-3ff4-44af-9f66-01d78ee8160e", "historyId": "469e1c7a7a786adeb6bc7642147fd103", "testCaseId": "469e1c7a7a786adeb6bc7642147fd103", "fullName": "tests.test_web_ui.TestLogin#test_failed_login", "labels": [{"name": "feature", "value": "登录功能"}, {"name": "story", "value": "失败登录"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_web_ui"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "P14s"}, {"name": "thread", "value": "23916-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_web_ui"}]}